install.sh

```diff
@@ -16,9 +16,21 @@ if [ "$1" = "install" ]; then
     if [[ ":$PATH:" != *":$script_dir:"* ]]; then
         echo "Adding $script_dir to PATH"
         # Add script_dir to PATH
-        echo "export PATH=\"\$PATH:$script_dir\"" >> ~/.bashrc
-        # Reload .bashrc
-        source ~/.bashrc
+
+        if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
```suggestion
        current_shell_path="$(echo $SHELL)"
        current_shell=$(basename $current_shell_path)"
        if [[ "$current_shell" == "zsh" ]]; then
```

Reply:
Are you sure that the $SHELL variable is set to zsh when running this script? Because if you look at the first line of this script you will see #/bin/bash

Reply:
I see the error I did, the script is always executed as bash script because the first line tell it to be.
Maybe I can just add the path each .rc files that exist in the user directory.